internal/app/knitgateway/version.go

```diff
@@ -21,22 +21,20 @@ import (
 	"connectrpc.com/connect"
 )
 
-const buildVersion = "v0.1.0-dev"
-
 //nolint:gochecknoglobals
 var (
-	// NB: This is a var instead of a const so it can be changed via -X ldflags.
+	// NB: These are vars instead of consts so they can be changed via -X ldflags.
+	buildVersion       = "v0.1.0"
```

Comment:
This needs to be a var not a const. Not only is this now referenced in the `Makefile`, but even before (when I first moved it to a const), it was actually set from goreleaser config. So in order for that to work, it must be a var. (Ideally, it's okay if it's a const and is not updated by an -X ldflag since it _should_ always be in sync with any actual release tag. But this is "just in case"...)

Reply:
Also, this is the key change, though it's perhaps non-obvious due to the code motion -- `v0.1.0-dev` -> `v0.1.0`.